playbooks/byo/openshift-prometheus/README.md

```diff
@@ -0,0 +1,131 @@
+OpenShift Prometheus
```

Comment:
I'm a bit confused. From where is this cherry-picked? There is no playbooks/byo/openshift-prometheus/ in master or 3.7.
In master and 3.7 the README.md is only inside the role and not around the playbook.

In 3.7 openshift-prometheus playbook is under byo/openshift-cluster.
I don't understand why the prometheus playbooks for 3.7 need to change in order to support grafana?

Reply:
its not like it need to change. We have it as Tech-Prev, and our upstream example moved a little bit. My customer needs "new features" from upstream in their Tech-Preview deployment.
+ I'm adding entry points (playbooks) into 3.7 branches as despite the fact we had a role in the roles folder - it was kinda "DIY" when it came to run it...

Reply:
3.7 already has the entry points in the playbook:
https://github.com/openshift/openshift-ansible/blob/release-3.7/playbooks/common/openshift-cluster/openshift_prometheus.yml.
I didn't argue the roles additions. just the playbook.
What do changes in the playbook area add?

Reply:
Playbooks entry in `byo` just makes things easy to understand for playbook consumer. As most of the playbooks follow same pattern. `byo` sets facts and calls `common`. 

I see I replicated whats in `common/openshift-cluster/openshift_prometheus.yml` and just created a separate dedicated folder (my bad). I think best way here would be to point `byo` to existing entry point in `openshift-cluster` so we would not duplicate them.